WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewController.swift

```diff
@@ -31,6 +31,25 @@ final class HubMenuViewController: UIHostingController<HubMenu> {
     func showPaymentsMenu() {
         show(InPersonPaymentsMenuViewController(), sender: self)
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        // We want to hide navigation bar *only* on HubMenu screen. But on iOS 16, the `navigationBarHidden(true)`
+        // modifier on `HubMenu` view hides the navigation bar for the whole navigation stack.
+        // Here we manually hide or show navigation bar when entering or leaving the HubMenu screen.
+        if #available(iOS 16.0, *) {
```

Comment:
do you still see the navigation bar issue https://github.com/woocommerce/woocommerce-ios/issues/7091 with the latest Xcode 14? somehow I don't see this issue anymore on `trunk` using Xcode 14 in an iOS 16.0 simulator. I don't have an iOS 16 device though, might be worth testing on one to confirm if this workaround is still needed.

Reply:
Just tried to comment out `viewWillAppear` and `viewWillDisappear` - reproduced the issue on the device:

Payments | Reviews
--|--
![IMG_2289](https://user-images.githubusercontent.com/3132438/190339977-86f58498-0437-4493-93e5-fcdc243b207b.PNG)|![IMG_2290](https://user-images.githubusercontent.com/3132438/190339994-10bb1349-f973-4e52-833b-49645306db80.PNG)

Reply:
@jaclync Hmm, that's strange. I just run the trunk branch on an iPhone 14 using Xcode 14, and verified this is still an issue.

Reply:
Sounds like something odd on my end then, thanks for checking both!